content/docs/tutorials/get-started/data-versioning.md

```diff
@@ -198,49 +122,38 @@ Usually, we also want to `git commit` and `git push` the corresponding
 
 <details>
 
-### Expand to see what happened internally
+### 💡 Expand to see what happens internally
```

Comment:
```suggestion
### 💡 Expand to see what happens under the hood
```
Like the previous section like this one?

Reply:
yep